README.rst

```diff
@@ -62,7 +63,7 @@ Development version
 ===================
 
 Check your python version >= 3.6, and install pyqt5, as instructed above in the
-`Getting started`_ section.
+`Getting started`_ section above or `Running from source on old Linux`_ section below.
```

Comment:
For macOS I can add a section here. We can commit this without the section and I can add it...